sites/partners/cypress/e2e/default/09-lottery.spec.ts

```diff
@@ -0,0 +1,155 @@
+describe("Lottery Tests", () => {
+  before(() => {
+    cy.login()
+  })
+
+  after(() => {
+    cy.signOut()
+  })
+
+  it("can run through every lottery action", () => {
+    const uniqueListingName = Date.now().toString()
+
+    cy.visit("/")
+
```

Comment:
Nit: Could this be a helper function? Like minimalTestListing() with param to be lottery listing or not? Same question for the minimal application piece below

Reply:
Will do! Great feedback ty!